.github/workflows/ci.yml

```
@@ -21,9 +21,9 @@ defaults:
     shell: bash
 
 env:
-  java_version: 21
+  java_version: 25
   java_distribution: zulu
-  javadoc_version: 24 # newer than `java_version` for better javadoc
+  javadoc_version: 24
```

should we bump this too? or just use `java_version` in place of `javadoc_version`?
```suggestion
  javadoc_version: 25
```

---

With 25, if javadoc_version isn't specified it isn't happy.  I didn't test specifying 25, only 24.

---

I'm taking a look now, and will push to this branch

---

looks ok now